binderhub/index.html

```diff
@@ -29,18 +29,19 @@
                     </div>
 
                     <form id="build-form" class="form jumbotron">
-                        <h4>Build & Launch a Repository</h4>
+                        <h4>Build and Launch a Binder</h4>
```

Comment:
Ideally I'd like to keep this to just Repository for now, rather than 'Binder'. Both to keep it consistent with current website (which Jeremy Freeman asked us) and to avoid introducing new terminology.